move/operators/sources/operators.move

```diff
@@ -18,10 +20,46 @@ module operators::operators {
     /// The main `Operators` struct storing the capabilities and operator IDs.
     public struct Operators has key {
         id: UID,
-        operator_ids: vector<ID>,
+        // number of operators should small and under the Sui vector limits
```

Comment:
```suggestion
        // The number of operators are small in practice, and under the Sui object size limit, so a dynamic collection doesn't need to be used
```
Let's try to be more precise so it makes sense to someone reading this a year later